classes/hook_callbacks.php

```diff
@@ -16,6 +16,8 @@
 
 namespace local_envbar;
 
+require_once(__DIR__ . '/../lib.php');
```

Comment:
I would recommend changing this to exact path like 

`require_once($CFG->dirroot . '/local/envbar/lib.php');`

Reply:
Hi @dmitriim ,

I have updated the patch